rcore/rebbleos.c

```diff
@@ -37,3 +41,29 @@ uint8_t rebbleos_get_system_status(void)
 {
     return system_status.booted;
 }
+
+SystemSettings *rebbleos_get_settings(void)
+{
+    return &_system_settings;
+}
+
+void rebbleos_module_set_status(uint8_t module, uint8_t enabled, uint8_t error)
```

Comment:
This seems kind of weird to me, and I'm not sure that it will survive as an abstraction, but for now, it exists, so...